### PR TITLE
Warning platform apps

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-86401

## Description of changes being made

Just adding a warning to make sure operators upgrade platform apps before proceeding with Konvoy upgrade.

### Preview

http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Review by some-things — APPROVED** [review]

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md [hunk 1]
```diff
@@ -32,10 +32,13 @@ To use the CLI to enable or disable applications, see [Application Deployment](.
 
 ## Upgrade Platform applications from the CLI
 
-Platform applications are deployed and upgraded as a set for each cluster or workspace. For the management cluster or workspace, Platform applications are automatically upgraded with the [DKP upgrade](../../../../dkp-upgrade) procedure; no other steps are necessary for the management cluster or workspace. For attached/managed clusters or workspaces, you MUST manually upgrade Platform applications bundle.
+Platform applications are deployed and upgraded as a set for each cluster or workspace. For the management cluster or workspace, Platform applications are automatically upgraded with the [DKP upgrade](../../../dkp-upgrade) procedure; no other steps are necessary for the management cluster or workspace. For attached/managed clusters or workspaces, you MUST manually upgrade Platform applications bundle.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:35]:
```suggestion
The [DKP upgrade](../../../dkp-upgrade) process deploys and upgrades Platform applications as a bundle for each cluster or workspace. For the management cluster or workspace, DKP upgrade handles all Platform applications; no other steps are necessary to upgrade the Platform application bundle. However, for managed or attached clusters or workspaces, you MUST manually upgrade the Platform applications bundle with the following command.
```

Comment (Colin1841) [reply pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:35]: Fixed passive & made this flow better - I don't like how that sounded.

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md [hunk 2]
```diff
@@ -32,10 +32,13 @@ To use the CLI to enable or disable applications, see [Application Deployment](.
...
+
+<p class="message--warning"><strong>WARNING: </strong>If you are upgrading your Platform applications as part of the <a href="../../../dkp-upgrade">DKP upgrade</a>, upgrade your Platform applications on any additional Workspaces before proceeding with the Konvoy upgrade. Some applications in the previous release are not compatible with the <a href="../../../release-notes/">Kubernetes version</a> of this release, and upgrading Kubernetes is part of the DKP Konvoy upgrade process.
+</p>
 
 Upgrade all platform applications in the given workspace and its projects to the same version as platform applications running on the management cluster with this command:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:40]:
```suggestion
Upgrade all platform applications in the given workspace and its projects to the same version as platform applications running on the management cluster:
```

**Review by Colin1841 — APPROVED** [review]

**Comment by mwrightd2** [comment]
@gvasquezvargas once the jenkins commit is built, please go ahead and merge.

**Pull request merged**